Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM golang:1.6-alpine
+
+RUN mkdir -p /go/src/redoctober
+WORKDIR /go/src/redoctober
+
+# https://github.com/gliderlabs/docker-alpine/blob/master/docs/usage.md#disabling-cache
+RUN apk --no-cache add git openssl
+
+# build binary
+COPY . /go/src/redoctober
+RUN go get -d -v
+RUN go install -v
+
+# cleanup
+RUN apk del git
+RUN rm -rf /var/cache/apk/*
```

Comment:
so the thing is if you do these on separate lines its kinda pointless because the layers still exist for the image, which is why I do the approach to start from apline and install go, build the package, etc and purge the build deps all on one line